kinder/cmd/kinder/do/do.go

```diff
@@ -122,6 +123,11 @@ func NewCommand() *cobra.Command {
 			"If not set, kubeadm will automatically choose the kubeadm config version "+
 			"according to the Kubernetes version in use",
 	)
+	cmd.Flags().StringVar(
+		&flags.FeatureGate,
+		"kubeadm-feature-gate", "",
+		"the kubeadm feature-gate to be used for init, join and upgrade",
```

Comment:
it is a bit limiting in terms of UX.

that said:
- i don't see having a single test with kinder that tests multiple FGs.
- we don't have that many FGs in kubeadm anyway, currently.

so supporting a single feature gate for the time being seems fine to me.